src/components/EventManagement/EventActionItems/EventActionItems.spec.tsx

```
@@ -62,21 +63,32 @@ vi.mock('subComponents/SortingButton', () => ({
+          } else if (dataTestIdPrefix === 'filter') {
+            // Cycle through filter states: all -> pending -> completed -> all
+            const nextCount = filterClickCount + 1;
+            setFilterClickCount(nextCount);
+            const filterStates = ['pending', 'completed', 'all'];
+            const currentState = filterStates[(nextCount - 1) % 3];
+            onSortChange(currentState);
+          } else {
+            onSortChange('test-value');
+          }
+        }}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)